ROADMAP.md

```diff
@@ -0,0 +1,98 @@
+Ansible Roadmap
+=============
+This document is now the location for published Ansible Core roadmaps.     
+
+The roadmap will be updated by version. Based on team and community feedback, an initial roadmap will be published for a major or minor version (2.0, 2.1).  Subminor versions will generally not have roadmaps published.
+
```

Comment:
we've been calling the 1.x major and 1.x.x minor until now, but this is closer to what anyone else would expect